sql/src/test/java/io/crate/integrationtests/SysNodeResiliencyIntegrationTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import com.google.common.collect.Sets;
+import io.crate.action.sql.SQLAction;
+import io.crate.action.sql.SQLRequest;
+import io.crate.testing.SQLTransportExecutor;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.test.ESIntegTestCase;
+import org.elasticsearch.test.disruption.NetworkPartition;
+import org.elasticsearch.test.disruption.NetworkUnresponsivePartition;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.hamcrest.Matchers.*;
+
+@ESIntegTestCase.ClusterScope(numDataNodes = 1, numClientNodes = 2)
+public class SysNodeResiliencyIntegrationTest extends SQLTransportIntegrationTest {
+
+    public SysNodeResiliencyIntegrationTest() {
+        super(new SQLTransportExecutor(
```

Comment:
it seems the test doesn't use the `execute` methods which access the SQLTransportExecutor, is it still necessary to have this custom clientProvider?